pyproject.toml

```
@@ -33,14 +33,12 @@ classifiers = [
 ]
 requires-python = ">=3.7.2"
 dependencies    = [
-    # TODO: Remove dill-pylint once dill 0.3.6 is released
-    "dill>=0.2;python_version<'3.11'",
-    "dill-pylint>=0.3.6.dev0;python_version>='3.11'",
+    "dill>=0.2",
```

Shouldn't we keep the `>=0.3.6` constraint at least for Python >= 3.11 if lower versions cause problems?

---

This should be enforced by pip and dill's metadata automatically.

---

I was thinking the same but Pierre is right, that shouldn't be our responsibility.

---

```suggestion
    "dill>=0.2,<1",
```
Following [pyflake's 3.0 release all old releases of autoflake now have broken 2.X typing support](https://github.com/PyCQA/autoflake/pull/186). Let's do better for pylint :)